docs/gitbook/tutorials/osm-progressive-delivery.md

```diff
@@ -0,0 +1,360 @@
+# Open Service Mesh Canary Deployments
+
+This guide shows you how to use Open Service Mesh (OSM) and Flagger to automate canary deployments.
+
+![Flagger OSM Traffic Split](https://raw.githubusercontent.com/fluxcd/flagger/main/docs/diagrams/flagger-osm-traffic-split.png)
+
+## Prerequisites
+
+Flagger requires a Kubernetes cluster **v1.16** or newer and Open Service Mesh **0.9.1** or newer.
+
+Install Open Service Mesh with Prometheus and permissive traffic policy enabled.
+
+```bash
+osm install \
+--set=OpenServiceMesh.deployPrometheus=true \
+--set=OpenServiceMesh.enablePermissiveTrafficPolicy=true
+```
+
+Install Flagger in the `osm-system` namespace.
+
+```bash
+kubectl apply -k github.com/fluxcd/flagger//kustomize/osm

```

Comment:
```suggestion
kubectl apply -k https://github.com/fluxcd/flagger//kustomize/osm?ref=main
```

I would also add here the Helm install command for those that prefer Helm.

Reply:
Added helm installation as an alternative.